src/pacasam/extractors/extractor.py

```diff
@@ -18,6 +17,8 @@ class Extractor:
     if they do not yet exist, and extraction operations are atomic at the patch level.
     """
 
+    patch_suffix: str
```

Comment:
is it ok not to initialize patch_suffix? Maybe having "" by default would be better (in case a child class doesn't override it)

Reply:
I think this is the interest of not initializing it: throwing an exception in case it was not set, so that the user is forced to specify the extension.